cob_default_robot_config/cob4-2/command_gui_buttons.yaml

```diff
@@ -71,8 +71,7 @@ group17:
   group_name: torso
   component_name: behavior
   buttons: [[torso_front,trigger,torso_front],
-    [torso_front_home,trigger,torso_front_home],
-    [pick,trigger,pick]]
+    [torso_front_home,trigger,torso_front_home]]
```

Comment:
maybe we can keep pick after https://github.com/ipa320/cob_robots/pull/411 is merged